packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -43,35 +57,41 @@ export function IntegrationList() {
       >
         {intl.formatMessage(integrationMessages.pageIntroduction)}
         <ListViewSimplified>
-          <ListViewItemSimplified
-            actions={
-              <>
-                <Pill label="Active" type="active" />
-                <ToggleMenu
-                  id="toggleMenu"
-                  menuItems={[
-                    {
-                      handler: () => {},
-                      label: intl.formatMessage(integrationMessages.revealKeys)
-                    },
-                    {
-                      handler: () => {},
-                      label: intl.formatMessage(integrationMessages.disable)
-                    },
-                    {
-                      handler: () => {},
-                      label: intl.formatMessage(integrationMessages.delete)
-                    }
-                  ]}
-                  toggleButton={<VerticalThreeDots />}
-                />
-              </>
-            }
-            label="Sweet Health"
-            value="Health Integration"
-          />
+          {offlineData.integrations.map((integration) => (
+            <ListViewItemSimplified
+              actions={
+                <>
+                  <Pill label="Active" type="active" />
```

Comment:
This should be conditional enable / disable depending on the status. + Translation